src/elements/ngm-cam-configuration.ts

```diff
@@ -253,12 +264,35 @@ export class NgmCamConfiguration extends LitElementI18n {
     }
   }
 
+  onClose() {
+    this.dispatchEvent(new CustomEvent('close'));
+    if (this.lockType) {
+      // keep commented tags to have translations
+      // i18next.t('cam_lock_info_elevation') i18next.t('cam_lock_info_angle')
+      // i18next.t('cam_lock_info_pitch') i18next.t('cam_lock_info_move')
+      showSnackbarInfo(i18next.t(`cam_lock_info_${this.lockType}`),
+        {
+          displayTime: 0,
```

Comment:
If the info should be displayed `forever`, I would rather pass `-1`, or eventually not `Number.MAX_VALUE`.

Reply:
according to https://fomantic-ui.com/modules/toast.html#/settings it should be 0